src/components/map/MapService.js

```diff
@@ -562,6 +570,17 @@ goog.require('ga_urlutils_service');
                     '/home.html'
               };
 
+              // Luftfahrthindernis
+              if (response.data['ch.bazl.luftfahrthindernis']) {
+                response.data['ch.bazl.luftfahrthindernis'].config3d =
+                    'ch.bazl.luftfahrthindernis.3d';
+                response.data['ch.bazl.luftfahrthindernis.3d'] = {
+                  type: 'kml',
+                  url: 'https://dav0.bgdi.admin.ch/bazl_web/bern/' +
+                      'Active_Obstacles_18.04.2016.bern.kml'
+                };
+              }
+
```

Comment:
This KML is a subset of the original one (only bern region) that I've made myself. It's not the official one.